app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        cmd, original, file_path = command.split()
```

Comment:
Don't split `command` a few times. Create a separate variable for storing result of `command.split()` and use it where you need